### PR TITLE
reversed order matching operation bindings are searched so custom ope…

### PR DESCRIPTION
**Pull request by fil512**
…rations override builtin ones

### CODE REVIEW

File: hapi-fhir-server/src/main/java/ca/uhn/fhir/rest/server/ResourceBinding.java [hunk 1]
```diff
@@ -35,7 +35,7 @@ public class ResourceBinding {
 	private static final org.slf4j.Logger ourLog = org.slf4j.LoggerFactory.getLogger(ResourceBinding.class);
 
 	private String resourceName;
-	private List<BaseMethodBinding<?>> myMethodBindings = new ArrayList<>();
+	private LinkedList<BaseMethodBinding<?>> myMethodBindings = new LinkedList<>();
```
Comment (jamesagnew) [inline hapi-fhir-server/src/main/java/ca/uhn/fhir/rest/server/ResourceBinding.java:38]: Meta-comment - This should have a test.

**Review by jamesagnew — CHANGES_REQUESTED** [review]

File: hapi-fhir-server/src/main/java/ca/uhn/fhir/rest/server/ResourceBinding.java (hunk 1 above)
Comment (fil512) [reply hapi-fhir-server/src/main/java/ca/uhn/fhir/rest/server/ResourceBinding.java:38]: I added a test in cdr.  But good point, I will add one here as well.

Comment (fil512) [reply hapi-fhir-server/src/main/java/ca/uhn/fhir/rest/server/ResourceBinding.java:38]: done

**Review by markiantorno — APPROVED** [review]

**Review by dmuylwyk — APPROVED** [review]
LGTM

**Pull request merged**